core/src/test/java/io/grpc/CallOptionsTest.java

```diff
@@ -150,15 +150,19 @@ public void toStringMatches_withDeadline() {
   }
 
   @Test
-  @SuppressWarnings("deprecation")
+  @Deprecated
```

Comment:
I don't think this is right.   `@Deprecated`  is for telling API users not to call the method, the suppress warnings is to tell the ide/compiler to just ignore it for now.  The latter is the right thing for a unit test.